modules/oe_editorial_corporate_workflow/src/Services/ShortcutTransitionHandlerInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_editorial_corporate_workflow\Services;
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Provides an interface for shortcut handler service.
+ */
+interface ShortcutTransitionHandlerInterface {
+
+  /**
+   * Method to handle revisions when shortcuts are used on the moderation form.
```

Comment:
```
Saves all the revisions for the transitions between two states.

This is needed because we do not want to skip the creation of the revisions that would have otherwise been created had we not used the shortcuts to skip transitions.
```